src/js/components/purchaseOrder/PurchaseOrderListStyles.scss

```diff
@@ -80,7 +80,6 @@
       }
     }
   }
-  // TODO: Delete this after table's restyling is ready
   .ReactTable .rt-tfoot .rt-td {
     border: 0 !important;
     font-weight: 600;
```

Comment:
I think you can delete this code, as TODO was written to delete this part as soon as your styling comes unless you use it in some way?